pkg/store/scan_test.go

```diff
@@ -124,16 +124,16 @@ func TestIScan(t *testing.T) {
 	st, closer := makeStore()
 	defer closer()
 
+	var err error
 	var zero []byte
 	zero = append(zero, byte(0))
-	idx, err := st.Set(schema.KeyValue{Key: zero, Value: []byte(`item0`)})
```

Comment:
I think error validation is worth here

Reply:
Please double-check this. If I understand it correctly, this first call is supposed to return an error, whereas the rest of the `st.Set` calls are supposed to succeed.

Reply:
Indeed, that's because first byte being 0 is currently reserved, it's a limitation that will be fixed. So for the scope and intent of this unit test, please feel free to remove that initial insertion. First statement (after store initialization) could be  idx, err := st.Set(schema.KeyValue{Key: []byte(`0`), Value: []byte(`itemZERO`)})

Reply:
last detail on this:
- lines 127-128 could be become useless, so can be removed as well.

Reply:
You're right. Done.